packages/storybook/src/stories/ino-chip/ino-chip.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { expect, Locator, test } from '@playwright/test';
+import { goToStory, setAttribute } from '../test-utils';
+
+test.describe('ino-chip', () => {
+  let chip: Locator;
+
+  test.beforeEach(async ({ page }) => {
+    await goToStory(page, ['buttons', 'ino-chip', 'default']);
+    chip = page.locator('ino-chip');
+  });
+
+  test('should render chip', async () => {
+    await expect(chip).toBeVisible();
+  });
+
+  test('should not render remove icon when removable is not set', async () => {
+    await expect(chip.getByRole('button')).toHaveCount(1);
```

Comment:
can't you just check for the chip locator?

```suggestion
    await expect(chip).toHaveCount(1);
```

Reply:
I think not. The chip selector access the `ino-chip`. With `chip.getByRole('button')` I access all elements with role `button` inside the `ino-chip` element.